nf_core/pipeline-template/lib/NfcoreTemplate.groovy

```diff
@@ -18,6 +18,20 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    //  Warn if a -profile or Nextflow config has not been provided to run the pipeline
+    //
+    public static void checkConfigProvided(workflow, log) {
+        if (workflow.profile == 'standard' && workflow.configFiles.size() <= 1) {
+            log.warn "[$workflow.manifest.name] You are attempting to run the pipeline without any custom configuration!\n\n" +
+                    "This will be dependent on your local compute enviroment but can be acheived by:\n" +
```

Comment:
one or more of?

Reply:
Added inhttps://github.com/nf-core/tools/pull/1309/commits/89af1d984b3eeefe3d8a985bcfe3abf4ee0d05e8